sound/soc/codecs/max98373.c

```diff
@@ -64,7 +64,7 @@ static const struct snd_kcontrol_new max98373_spkfb_control =
 static const struct snd_soc_dapm_widget max98373_dapm_widgets[] = {
 SND_SOC_DAPM_DAC_E("Amp Enable", "HiFi Playback",
 	MAX98373_R202B_PCM_RX_EN, 0, 0, max98373_dac_event,
-	SND_SOC_DAPM_POST_PMU | SND_SOC_DAPM_POST_PMD),
+	SND_SOC_DAPM_POST_PMU | SND_SOC_DAPM_PRE_PMD),
```

Comment:
and you didn't reply to this comment

Reply:
@LongWang100 is this because if you first disable the PCM input, somehow the amplifier may work with data that has a discontinuity?

It's kind of strange, usually the sequence is to stop taking data into account - but presumably do so on a zero crossing - and only power-down later.

But if this is experimentally confirmed, then that's fine with me. It's not clear from the commit message whether this is a tentative fix or a real one that's been verified on actual hardware.

Reply:
I don't think so. The pop here is produced by the analog domain. So we power down the amplifier first to avoid hearing the pop. It is confirmed in the Chromebook project(Kano).